### PR TITLE
yet another round of pico typo fixes

### PR DESCRIPTION
**Pull request by AntonioMeireles**
Hey All, 

because it's Friday, 😄 

Keep the great work && a great weekend!

### CODE REVIEW

File: cookbook/File_Upload.md [hunk 1]
```diff
@@ -216,7 +216,7 @@ const ImageForm = (props) => {
   />
 ```
 
-Well lookie here:
+Well, look here:
```
Comment (jtoar) [inline cookbook/File_Upload.md:219]: Ok if we preserve Rob's humor here or did you find it confusing?

File: docs/webpackConfiguration.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ yarn rw build --stats
 
 Note that this'll skip building the api side.
 
-This config uses [Webpack Bundle Analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer). When it finishes, it'll launch an interactive zoomable, treemap in your browser to examine the contents of all your bundles.
+This config uses [Webpack Bundle Analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer). When it finishes, it'll launch an interactive zoomable, tree-map in your browser to examine the contents of all your bundles.
```
Comment (jtoar) [inline docs/webpackConfiguration.md:62]: Webpack uses "treemap" all over their docs so I feel like this one is ok.

**Review by jtoar — COMMENTED** [review]
Thanks @AntonioMeireles, always amazed at how many typos slip through so really grateful for these! Agree with most of the fixes, just had comments about two of them, let me know what you think.

File: cookbook/File_Upload.md (hunk 1 above)
Comment (AntonioMeireles) [reply cookbook/File_Upload.md:219]: well, it's an idiomatic thing, and i guess that as time goes by most of the audience will end being non native english speakers.. so some linguistic _metadata_ IMHO ends lost over the wire. i guess that a middle ground would be to start using as a convention italics when we actually intend to spell something in a non canonical way. will do that in next force push if you don't mind.

File: docs/webpackConfiguration.md (hunk 2 above)
Comment (AntonioMeireles) [reply docs/webpackConfiguration.md:62]: @jtoar 

gotcha. will also italicise it if you don't mind.

**Comment by AntonioMeireles** [comment]
@jtoar 

just pushed again. see if palatable now... if not i'll just revert those two mods. 

All the best && thanks for your patience

António

File: cookbook/File_Upload.md (hunk 1 above)
Comment (cannikin) [reply cookbook/File_Upload.md]: My humor transcends all language barriers! :)

My only issue with italics is that I use them just for emphasis of certain words to mimic speech, and in this case I wouldn't emphasize "lookie" in that context.

Comment (AntonioMeireles) [reply cookbook/File_Upload.md]: @cannikin (hi!), @jtoar 

so, how do we proceed here ?

just drop this mod, whatever the form ?

Comment (AntonioMeireles) [reply cookbook/File_Upload.md]: well, just dropped it in latest force-push. it's friday after all :-)

**Pull request merged**